docs/content/en/docs/concepts/architecture/components/metrics-operator/_index.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Keptn Metrics Operator
+linktitle: Metrics Operator
+description: Basic understanding of Keptn's Metrics Operator
+weight: 80
+cascade:
+---
+
+
+**Metrics Operator** collects, processes, and analyzes metrics data from a variety of sources. Once collected, this data, can be used to generate a variety of reports and dashboards that provide insights into the health and performance of the application and infrastructure.
+
```

Comment:
I took these notes of information that needs to be included.  Alas, my notes are a bit garbled -- need @thisthat and @bacherfl to review, correct, and clarify the second paragraph.  I think the first paragraph of what I have fits nicely here -- your nice definition of what the Keptn Metrics Operator is, then mention that Kubernetes does have metric operators but they have limitations...

Kubernetes has two metrics servers,
one is custom and the other is external.
However, neither of these APIs allow you to use multiple observability platforms
such as Prometheus, Dynatrace, or Datadog;
you must select only one of them.

The Keptn Metrics Operator provides a single entry point
that allows you to provide all your data [and here my notes get garbled]

The Keptn Metrics Operator is HBA compatible so that the workload can be scaled horizontally

Reply:
Agree, I think the most important thing to highlight here is that the metrics operator provides a common interface for consuming metrics. I.e. the values of those metrics can all be retrieved via the KeptnMetric CRD, regardless of which observability platform these values come from.
Due to the integration with the Kubernetes custom metrics API, these metrics are also compatible with the Kubernetes HorizontalPodAutoscaler (HPA)